source/_integrations/nest.markdown

```diff
@@ -466,6 +466,8 @@ Once you have completed the above steps, you can continue through the flow to re
 
 ## Troubleshooting
 
+- *No access to partner information* "Infomation could not be retrieved" message during the setup wizard, please ensure that you have successfully migrated your Nest Account to a Google Account using the Google Nest App. Additionally, if your home has multiple members, please note that the individual who initially set up the home must complete the migration of their Nest Account to a Google Account before you can establish a connection with Home Assistant.
```

Comment:
I appreciate adding the documentation, thank you.

My impression is that this error message means that the account is not able to access the Home, and there are multiple reasons for this -- where the issue you'e cited here is probably one of the most common. I've made a slight adjustment to give a little more context.


```suggestion
- *No access to partner information* "Infomation could not be retrieved" message during the setup wizard means that the Google Account used is not able to access the Home.  Please ensure that you have successfully migrated your Nest Account to a Google Account using the Google Nest App. Additionally, if your home has multiple members, please note that the individual who initially set up the home must complete the migration of their Nest Account to a Google Account before you can establish a connection with Home Assistant.
```

We can also link to the nest docs here as well.
https://developers.google.com/nest/device-access/reference/errors/pcm#no_access_to_partner if that would be helpful.

Reply:
Sorry added a typo correction before seeing your suggestion. Will incorporate your edit now.